devices/heiman.js

```diff
@@ -737,6 +737,20 @@ module.exports = [
         },
         exposes: [e.battery(), e.action(['pressed']), e.battery_low(), e.tamper()],
     },
+    {
+        fingerprint: [{modelID: 'DoorBell-EF-3.0', manufacturerName: 'HEIMAN'}],
+        model: 'HS2SS-E_V03',
```

Comment:
Is this the correct model? https://www.heimantech.com/manual/index.php?type=detail&id=44 says this is a smart scene switch.